src/test/java/com/google/sps/UtilityTest.java

```diff
@@ -130,6 +136,31 @@ public void tryParseInt_nullValue_returnsZero() {
     Assert.assertEquals(actual, expected);
   }
 
+  /** Tests for getUsersToNotify(). */
+  @Test
+  public void getUserToNotifyQuestion() {
```

Comment:
A common style practice for naming java unit test functions it to include the method being tested, the test condition, and the expected outcome, so something like: getUsersToNotify_Question_Success() or getUsersToNotify_NoType_ReturnsEmpty()

Doing this makes it really explicitly clear what is being tested, how it's being tested, and what we expect to see. Consider updating your methods to also follow this pattern!

It's not technically part of the public github Java style guide, so I'll leave it up to you - but a good habit to have :)

Reply:
Done!